<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>728</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_draw"/>
   <addaction name="action_plus"/>
   <addaction name="action_minus"/>
   <addaction name="separator"/>
  </widget>
  <action name="action_plus">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/plus.png</normaloff>:/images/plus.png</iconset>
   </property>
   <property name="text">
    <string>plus</string>
   </property>
  </action>
  <action name="action_minus">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/minus.png</normaloff>:/images/minus.png</iconset>
   </property>
   <property name="text">
    <string>minus</string>
   </property>
  </action>
  <action name="action_draw">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/draw.png</normaloff>:/images/draw.png</iconset>
   </property>
   <property name="text">
    <string>draw</string>
   </property>
  </action>
  <action name="action_calculate">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/calculator_2.png</normaloff>:/images/calculator_2.png</iconset>
   </property>
   <property name="text">
    <string>calculate</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
